ethportal-api/src/types/execution/header.rs

```diff
@@ -262,20 +262,21 @@ impl From<Header> for RpcHeader {
             receipts_root,
             logs_bloom,
             difficulty,
-            number: Some(U256::from(number)),
-            gas_limit,
-            gas_used,
-            timestamp: U256::from(timestamp),
+            number,
+            gas_limit: gas_limit.to::<u128>(),
```

Comment:
nit: I didn't check it, but I think you don't have to specify `::<u128>` as compiler should be able to figure it out. But it's fine if you want to be explicit. Here and in most other places